packages/subgraph/src/permissions.mapping.ts

```diff
@@ -195,7 +195,7 @@ export function handleScopeParameter(event: ScopeParameter): void {
   const parameter = new Parameter(parameterId) // will always overwrite the parameter
   const paramType = PARAMETER_TYPE[event.params.paramType]
   const paramComp = PARAMETER_COMPARISON[event.params.paramComp]
-  const compValue = event.params.compValue
+  const compValue = event.params.compValue // Can't be decode here, we do not have the required info (target ABI), must be done in frontend.
```

Comment:
Could you use something like https://www.4byte.directory or https://sig.eth.samczsun.com/ to decode here?

Reply:
I don't think it's a problem since we can do it more reliably in the frontend.

Also, we can't do any external requests here (in AssemblyScript). We could, however, perhaps use something like this to guess the ABI if there is no.